workload_profiling/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/ProfileUtils.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.tool.profiling
+
+import org.apache.spark.internal.config
+import org.apache.spark.sql.SparkSession
+
+/**
+ * object Utils provides toolkit functions
+ *
+ */
+object ProfileUtils {
+
+  // Create a SparkSession in local mode
+  def createSparkSession: SparkSession = {
+    SparkSession
+        .builder()
+        .master("local[*]")
```

Comment:
shouldn't specify master here anymore as launch would handle that

Reply:
Was mentioning it here for creating Sparksession for unit tests.

Reply:
change the unit test to call a different version that creates it with local mode